frontend/styles/var/colors.ts

```diff
@@ -3,7 +3,7 @@ export type Colors = keyof typeof COLORS;
 export const COLORS = {
   // core colors
   "core-fleet-black": "#192147",
-  "core-fleet-blue": "#6A67FE",
+  "core-fleet-blue": "#6A67FE", // TODO: Why does this match ui-vibrant-blue and not core-fleet-blue
```

Comment:
What do you mean?

Reply:
looks like the mapping of the names aren't correct, 
- core-fleet-blue <> tooltip-bg 
- core-vibrant-blue <> core-fleet-blue